### ISSUE TITLE
Kolibri not installed on every device

### ISSUE THREAD

**Issue by aronasorman** [learningequality/kolibri#1]
Branch: develop
OS: all OSes

Hi,

I just noticed that Kolibri isn't installed on my laptop as of this writing. I checked other PCs and it's not there as well. How can I get to learning if it's not installed on every PC or device I use?

Please fix.

**Comment by jamalex** [comment]
:heart:

**Comment by indirectlylit** [comment]
this still seems to be an issue

**Comment by jamalex** [comment]
Yeah, definitely work-in-progress, but I believe the team is focused on resolving it promptly.

My fridge, for instance, is still not running Kolibri...

**Comment by radinamatic** [comment]
We are dropping the ball on this one, issue's still there... :stuck_out_tongue_winking_eye:

**Comment by jamalex** [comment]
Do we have a target date on this? The number of devices keeps increasing, so it's better if we can tackle this one proactively and avoid accruing implementation debt.

**Comment by MisRob** [comment]
Oh that's brilliant :) Took me a while to understand how could I get here 😄 

I would propose changing
> Branch: master

to some of our release branches since we don't have `master` anymore!

**Comment by rtibbles** [comment]
I tried to make a PR over the weekend to fix this, but I couldn't get the tests to pass.

**Comment by jamalex** [comment]
Just a gentle nudge; it's been a couple years, and while we've seen progress on this, it's still not quite complete... Would be good to tick this box so we can move forward from there!

**Comment by Lewiscowles1986** [comment]
What is the actual issue? Is it OS integration or a joke?

**Comment by rtibbles** [comment]
It's a joke - but we appreciate any help in getting us closer to resolving it!

**Comment by MisRob** [comment]
What's the progress on this, team?

**Comment by rtibbles** [comment]
Our metrics indicate that we are definitely making progress - but we are still a long way off the success criteria here.

**Comment by rtibbles** [comment]
@marcellamaki have we considered how to spec this issue to allow us to incrementally tackle it? Maybe a task list would help?

**Comment by pranavmen** [comment]
heyy ! could I work on this

**Comment by MisRob** [comment]
Hey @pranavmen, what will be your strategy to resolve it?

**Comment by pranavmen** [comment]
thanks for noticing @MisRob 
We will set up one 'Learning Hub' on your network. Once that is done, every device you own can access the full library through a simple web link, saving you storage and setup time
why would it be helpful --> Instant Availability , Unified Progress and Low Maintenance

**Comment by MisRob** [comment]
@pranavmen I'm sorry, that won't do - this issue's purpose is to introduce Kolibri on every device in the world ;) (you can scroll up and read the comments - in one of them it's stated that this very first issue #1 is a little joke) In any case, thanks for the offer and happy new year!

**Comment by rtibbles** [comment]
@coderabbitai plan
